src/error-display/error-display-data.tsx

```diff
@@ -319,16 +328,16 @@ const ERROR_DISPLAY_DATA = new Map<ErrorDisplayType, ErrorDisplayDataAttrs>([
             img: imgAttributeHelper(ImgPaths["insufficient-credits"]),
             title: "Insufficient credits",
             description:
-                "When the user has insufficient credits for a transaction.",
+                "You do not have enough credits to make this transaction.",
         },
     ],
     [
         "inactivity",
         {
             img: imgAttributeHelper(ImgPaths["inactivity"]),
-            title: "Inactive state",
+            title: "Are you still there?",
             description:
-                "When the user has been inactive for a certain period of time, and will be logged out after X minutes.",
+                "You’ve been inactive for a while. To protect your privacy, you’ll be logged out in X minutes YY seconds.\n\nIf you wish to stay on this page, let us know now.",
```

Comment:
You could pass this as a JSX if it is easier. But this is fine too.

Reply:
i'm not sure how to pass the countdown variables as JSX >_< so i'll leave it like this as a placeholder